common/LongPathHandler.go

```diff
@@ -0,0 +1,56 @@
+package common
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+//ToExtendedPath converts short paths to an extended path.
+func ToExtendedPath(short string) string {
```

Comment:
Can you add  general comment please, to explain why we are not using the built-in Go long path support. E.g. "We are not using the built-in Go long path support (that was introduced to the Go SDK in commit 231aa9d6d7) for two reasons:
(a) the main reason is that it does not support UNC paths, and we want to support those as best we can. (we have a known limitation when globbing, see globWrapper(), but apart from that, we believe we fully support long UNC paths with the approach we have taken)
(b) the secondary reason is that the Go SDK fix did not cover the lower-level OS calls that we open files with. "